From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ram Muthiah <rammuthiah@google.com>
Date: Mon, 21 Oct 2019 11:48:25 -0700
Subject: ANDROID: Move from clang r353983c to r365631c

Bug: 139440459
Test: BUILD_CONFIG=common/build.config.gki.x86_64 build/build.sh
Test: BUILD_CONFIG=common/build.config.gki.aarch64 build/build.sh
Change-Id: I7ee5d9a687843c09bbc5edb591d3337bd5bfbcbc
Signed-off-by: Ram Muthiah <rammuthiah@google.com>
---
 build.config.common | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.config.common b/build.config.common
index 3f1fac231a82..a879f6282c0f 100644
--- a/build.config.common
+++ b/build.config.common
@@ -2,7 +2,7 @@ BRANCH=android-mainline
 KERNEL_DIR=common
 
 CC=clang
-CLANG_PREBUILT_BIN=prebuilts-master/clang/host/linux-x86/clang-r353983c/bin
+CLANG_PREBUILT_BIN=prebuilts-master/clang/host/linux-x86/clang-r365631c/bin
 BUILDTOOLS_PREBUILT_BIN=build/build-tools/path/linux-x86
 
 EXTRA_CMDS=''
